### ISSUE TITLE
Not able to launch anaconda navigator

### ISSUE THREAD

**Issue by ryang6** [ContinuumIO/anaconda-issues#1736]
Help! I am not able to launch anaconda navigator. I have tried both python 2.7 64bit and 32bit installer, the most recent version from the website, none of them let me to even open up anaconda navigator. The 32 bit version at least showed the green circle was initializing and then disappeared, but nothing happened when I click on the navigator of 64 bit version.  I have python 2.7.13 installed and the windows 8, 6.4 bit operating system.

**Comment by goanpeca** [comment]
@ryang6 could you open the `Anaconda Prompt` and type

`anaconda-navigator` and copy what the console outputs?

**Comment by ryang6** [comment]
Traceback (most recent call last):
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\Scripts\conda-script.py
", line 5, in <module>
    sys.exit(conda.cli.main())
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\cli\main.py", line 151, in main
    activate.main()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\cli\activate.py", line 167, in main
    import conda.install
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\install.py", line 37, in <module>
    from .core.package_cache import rm_fetched  # NOQA
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\core\package_cache.py", line 10, in <module>
    from .path_actions import CacheUrlAction, ExtractPackageAction
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\core\path_actions.py", line 31, in <module>
    from ..gateways.download import download
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\gateways\download.py", line 16, in <module>
    from ..connection import CondaSession
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\conda
\connection.py", line 55, in <module>
    python=platform.python_implementation(),
  File "C:\Python27\Lib\platform.py", line 1460, in python_implementation
    return _sys_version()[0]
  File "C:\Python27\Lib\platform.py", line 1422, in _sys_version
    repr(sys_version))
ValueError: failed to parse CPython sys.version: '2.7.13 |Anaconda 4.3.1 (32-bit
)| (default, Dec 19 2016, 13:36:02) [MSC v.1500 32 bit (Intel)]'

C:\Users\RYANG>anaconda-navigator
An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



`$ C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\Scripts\conda-script.py info
 --json`




    Traceback (most recent call last):
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\exceptions.py", line 573, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\cli\main.py", line 98, in _main
        imported = importlib.import_module(module)
      File "C:\Python27\Lib\importlib\__init__.py", line 37, in import_module
        __import__(name)
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\cli\main_list.py", line 20, in <module>
        from ..egg_info import get_egg_info
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\egg_info.py", line 15, in <module>
        from .misc import rel_path
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\misc.py", line 19, in <module>
        from .core.index import get_index, _supplement_index_with_cache
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\core\index.py", line 8, in <module>
        from .package_cache import PackageCache
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\core\package_cache.py", line 10, in <module>
        from .path_actions import CacheUrlAction, ExtractPackageAction
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\core\path_actions.py", line 31, in <module>
        from ..gateways.download import download
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\gateways\download.py", line 16, in <module>
        from ..connection import CondaSession
      File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\c
onda\connection.py", line 55, in <module>
        python=platform.python_implementation(),
      File "C:\Python27\Lib\platform.py", line 1460, in python_implementation
        return _sys_version()[0]
      File "C:\Python27\Lib\platform.py", line 1422, in _sys_version
        repr(sys_version))
    ValueError: failed to parse CPython sys.version: '2.7.13 |Anaconda 4.3.1 (32
-bit)| (default, Dec 19 2016, 13:36:02) [MSC v.1500 32 bit (Intel)]'

2017-05-17 11:06:42,799 - WARNING conda_api.communicate:199
Conda command output on stderr

Traceback (most recent call last):
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\Scripts\anaconda-naviga
tor-script.py", line 5, in <module>
    sys.exit(anaconda_navigator.app.main.main())
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\app\main.py", line 52, in main
    start_app(options)
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\app\start.py", line 87, in start_app
    window = run_app(splash)
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\app\start.py", line 42, in run_app
    window = MainWindow(splash=splash)
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\widgets\main_window.py", line 162, in __init__
    self.api = AnacondaAPI()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\api\anaconda_api.py", line 718, in AnacondaAPI
    ANACONDA_API = _AnacondaAPI()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\api\anaconda_api.py", line 50, in __init__
    self._conda_api = CondaAPI()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\api\conda_api.py", line 1302, in CondaAPI
    CONDA_API = _CondaAPI()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\api\conda_api.py", line 307, in __init__
    self.set_root_prefix()
  File "C:\Users\RYANG\AppData\Local\Continuum\Anaconda2\lib\site-packages\anaco
nda_navigator\api\conda_api.py", line 427, in set_root_prefix
    self.ROOT_PREFIX = info['root_prefix']
TypeError: string indices must be integers, not str

**Comment by goanpeca** [comment]
@kalefranz whats this:
```
File "C:\Python27\Lib\platform.py", line 1460, in python_implementation
return _sys_version()[0]
File "C:\Python27\Lib\platform.py", line 1422, in _sys_version
repr(sys_version))
ValueError: failed to parse CPython sys.version: '2.7.13 |Anaconda 4.3.1 (32-bit
)| (default, Dec 19 2016, 13:36:02) [MSC v.1500 32 bit (Intel)]'
```
Have you seen that before?

**Comment by ryang6** [comment]
What does this error message mean? I saw a couple of "failed to", but I don't remember which one. I do remember I was not able to install pandas through pip. Is something related to Anaconda error?

**Comment by mingwandroid** [comment]
It means you have a messed up system. Please open the `Anaconda Prompt` from the `Start Menu` and try from there. Then, from the same prompt do `echo %PATH%` and paste the results.

**Comment by ameliawilson** [comment]
Where do you open the Anaconda Prompt?
Today I tried to reinstall Anaconda. The navigator will not open. in the terminal I tried "anaconda-navigator
-bash: anaconda-navigator: command not found"

**Comment by ryang6** [comment]
What I pasted here was from the Anaconda Prompt that open from the start Memu. Did I do something wrong.

**Comment by ameliawilson** [comment]
ryang6 I am not sure. I have the same issue and am also trying to figure it out.

**Comment by kalefranz** [comment]
please show us the output of

    echo %PATH%

**Comment by ryang6** [comment]
C:\Users\RYANG>echo %path%
C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\Intel\iCLS Client\;C:
\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\Syst
em32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Intel\Int
el(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Managemen
t Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine
Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Component
s\IPT;C:\Program Files\SASHome\Secure\ccme4;C:\Program Files\SASHome\x86\Secure\
ccme4;C:\Python27;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program
Files\nodejs\;C:\Program Files (x86)\Git\cmd;C:\Python27\;C:\Python27\Scripts;C:
\Program Files (x86)\sbt\bin;C:\Program Files\Java\jdk1.8.0_112\bin;C:\Program F
iles (x86)\Boundless\OpenGeo\jetty\jre\bin;C:\Program Files (x86)\Boundless\Open
Geo\bin;C:\Program Files (x86)\Boundless\OpenGeo\sdk\bin;C:\Program Files (x86)\
Boundless\OpenGeo\pgsql\bin;C:\Users\RYANG\AppData\Local\Continuum\Anaconda2;C:\
Users\RYANG\AppData\Local\Continuum\Anaconda2\Scripts;C:\Users\RYANG\AppData\Loc
al\Continuum\Anaconda2\Library\bin;C:\Program Files (x86)\Intel\iCLS Client\;C:\
Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\Syste
m32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Intel\Inte
l(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management
 Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine C
omponents\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components
\IPT;C:\Program Files\SASHome\Secure\ccme4;C:\Program Files\SASHome\x86\Secure\c
cme4;C:\Windows\system32\config\systemprofile\.dnx\bin;C:\Program Files\Microsof
t DNX\Dnvm\;C:\Python27\Scripts;C:\Users\RYANG\AppData\Roaming\npm;C:\Program Fi
les (x86)\Java\jre7\bin

**Comment by kalefranz** [comment]
From https://github.com/conda/conda/issues/4901#issuecomment-287868746

> Here you have two different Python installations and they are stepping on each other's toes.
> 
> I am guessing you elected to allow the installer to add itself to your system `PATH` but already had 'normal' Python 2.7 installed to the default location and also present in your `PATH`?
> 
> Unfortunately our installer adds its entries to the end of `PATH` so DLLs for other software frequently get loaded instead of ours (Python itself consists of many DLLs).
>
> I think, if the Anaconda Installer is going to add to `PATH` it should add its entries to the front of `PATH` like nearly all other installers do, but I more strongly think that software should not add itself to `PATH` by default (or recommend that to users).
>
> If you use the `Anaconda Prompt` to launch your command prompt then that will add the correct entries to the front of your `PATH` temporarily (i.e. for that command prompt instance only).

**Comment by ameliawilson** [comment]
@kalefranz I see this is closed, and I have tried to follow the similar posts and I still cannot open Anaconda Navigator. I am on MacOS 10.10.5. I originally had Anaconda with Python 2.7, I tried to uninstall it then install the one with Python 3.6. This is when everything got messed up and I cannot open the navigator. 

**I see it is most likely a path problem but I have no idea what to change to fix it**
This is what I have tried in the terminal. 
[terminal.txt](https://github.com/ContinuumIO/anaconda-issues/files/1048438/terminal.txt)

And this is the path

'# added by Anaconda3 2.3.0 installer
export PATH="/Applications/anaconda/bin:$PATH"'
